gateway/1.4/modules/ROOT/pages/_partials/logging-severity-level.adoc

```diff
@@ -0,0 +1,67 @@
+[[severity-levels]]
+== Message Logging Severity Levels
+
+When you configure a Message Logging policy for your API instance you can assign a severity level. 
+
+=== Severity Levels Description
+
+Each severity level includes a specific set of information in the log report. The following table
+describes the type of information that will be included in the logs for each severity level.
+
+[%header,cols='1a,5a'] 
+|===
+|Log Severity Level
+|Description
+
+|ERROR
+|Error events that prevent normal program execution but might allow the application to continue running.
+
+|WARN
+|Potentially harmful situations that indicate potential problems.
+
+|INFO
+|Informational messages that highlight the progress of the application.
+
+|DEBUG
+|Tracing information used by application developers.
+
+|===
+
+=== Severity Levels Hierarchy
+
+Log reports include only the severity level you select and the subsequent levels on the hierarchy .
```

Comment:
```suggestion
Log reports include only the selected severity level and the subsequent levels on the hierarchy.
```

Reply:
Is subsequent and hierarchy the right words here? Do you mean each level includes all the levels that follow?